docs/get-started/price-feeds/price-feeds.mdx

```diff
@@ -109,23 +110,24 @@ The following feeds are available on-chain and are compatible with the [Chainlin
+| BNB         | USD          | 0.1%                | 1m        | [0x8dd2D85C7c28F43F965AE4d9545189C7D022ED0e](https://bscscan.com/address/0x8dd2D85C7c28F43F965AE4d9545189C7D022ED0e#readProxyContract)  |
+| BTC         | USD          | 0.1%                | 1m        | [0xa51738d1937FFc553d5070f43300B385AA2D9F55](https://bscscan.com/address/0xa51738d1937FFc553d5070f43300B385AA2D9F55#readProxyContract)  |
+| weETH       | eETH         | 0.5%                | 6h        | [0x9b2C948dbA5952A1f5Ab6fA16101c1392b8da1ab](https://bscscan.com/address/0x9b2C948dbA5952A1f5Ab6fA16101c1392b8da1ab#readProxyContract)  |
+| STONE       | USD          | 0.5%                | 6h        | [0xa41107f9259bB835275eaCaAd8048307B80D7c00](https://bscscan.com/address/0xa41107f9259bB835275eaCaAd8048307B80D7c00#readProxyContract)  |
+| ETH         | USD          | 0.5%                | 6h        | [0x9cF19D284862A66378c304ACAcB0E857EBc3F856](https://bscscan.com/address/0x9cF19D284862A66378c304ACAcB0E857EBc3F856#readProxyContract)  |
+| USDC        | USD          | 0.5%                | 6h        | [0xeA2511205b959548459A01e358E0A30424dc0B70](https://bscscan.com/address/0xeA2511205b959548459A01e358E0A30424dc0B70#readProxyContract)  |
+| USDT        | USD          | 0.5%                | 6h        | [0xf57bA29437C130e155Ca4b65128630777638F05D](https://bscscan.com/address/0xf57bA29437C130e155Ca4b65128630777638F05D#readProxyContract)  |
+| SolvBTC     | USD          | 0.5%                | 6h        | [0xF5F641fF3c7E39876A76e77E84041C300DFa4550](https://bscscan.com/address/0xF5F641fF3c7E39876A76e77E84041C300DFa4550#readProxyContract)  |
+| wstETH      | ETH          | 0.5%                | 6h        | [0xE7e734789954e6CffD8C295CBD0916A0A5747D27](https://bscscan.com/address/0xE7e734789954e6CffD8C295CBD0916A0A5747D27#readProxyContract)  |
+| wstETH      | ETH\*        | 0.5%                | 6h        | [0xF43a01A39Fd28950031a42861faD703147d5C8e7](https://bscscan.com/address/0xF43a01A39Fd28950031a42861faD703147d5C8e7#readProxyContract)  |
+| wstETH      | stETH        | 0.5%                | 6h        | [0xa76dB2Cb356ba111cCB5a7Ca369D17E1592f42Dd](https://bscscan.com/address/0xa76dB2Cb356ba111cCB5a7Ca369D17E1592f42Dd#readProxyContract)  |
+| pufETH      | ETH          | 0.5%                | 6h        | [0xCa8b247304767619fCcF5b0829D0a5AAf414BA7D](https://bscscan.com/address/0xCa8b247304767619fCcF5b0829D0a5AAf414BA7D#readProxyContract)  |
+| SolvBTC.BBN | SolvBTC      | 0.5%                | 6h        | [0x24c8964338Deb5204B096039147B8e8C3AEa42Cc](https://bscscan.com/address/0x24c8964338Deb5204B096039147B8e8C3AEa42Cc#readProxyContract)  |
+| PUFFER      | USD          | 0.5%                | 6h        | [0x85C4F855Bc0609D2584405819EdAEa3aDAbfE97D](https://etherscan.io/address/0x85C4F855Bc0609D2584405819EdAEa3aDAbfE97D#readProxyContract) |
```

Comment:
the link leads to ehterscan and not bscscan and it leads to RedStone Price Feed for pumpBTC/BTC

Reply:
Good catch